pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -224,6 +324,36 @@ def level(
     num_noise_channels: int = 3,
     inplace: bool = True,
 ) -> SequentialWithOutChannels:
+    r"""Defines one level of the transformer from :cite:`ULVL2016`.
+
+    A level contains either only a
+    :class:`~pystiche_paper.ulyanov_et_al_2016._modules.ConvSequence` if it is the first
+    level or a :class:`~pystiche_paper.ulyanov_et_al_2016._modules.BranchBlock`
+    followed by a :class:`~pystiche_paper.ulyanov_et_al_2016._modules.ConvSequence`. The
+    :class:`~pystiche_paper.ulyanov_et_al_2016._modules.BranchBlock` gets
+    ``prev_level_block`` as ``intermediate`` in an
+    :class:`~pystiche_paper.ulyanov_et_al_2016._modules.HourGlassBlock` as
+    ``deep_branch`` and a
+    :class:`~pystiche_paper.ulyanov_et_al_2016._modules.ConvSequence` as
+    ``shallow_branch``.
```

Comment:
Too detailed. It should only state that

- level is a conv sequence
- if its not the first, the previous is put into hourglass block and joined with a branch block and finally followed by another conv sequence